base/cvd/BUILD.bazel

```diff
@@ -71,6 +71,17 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
+    name = "libffi_linux_arm64",
+    hdrs = [
+        "libffi/linux-arm64/ffi.h",
+        "libffi/linux-arm64/fficonfig.h",
+        "libffi/linux-arm64/ffitarget.h",
+    ],
+    includes = ["libffi/linux-arm64"],
+    visibility = ["//visibility:public"],
+)
```

Comment:
this should be in `libffi/linux-arm64/BUILD`

Reply:
Moved cc library declaration to `libffi/linux-arm64/BUILD`
Also moved x86 related build rules to `libffi/linux-x86/BUILD`